ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhoai-operator-interop__v4.14-GA-production.yaml

```diff
@@ -60,8 +60,41 @@ tests:
     test:
     - ref: rhoai-tests
     workflow: operator
+- as: smoke-ipi-aws-rhoai-baseline-fips
```

Comment:
these should be in a separate file, not `hypershift`-related

Reply:
@rnetser renamed files from `rhoai-addon` to use `rhoai-operator` branch and only updated AWS IPI and AWS OSD configs to have `operator` workflow for now, since we decided to first run jobs for this.

Reply:
Updated all jobs for 4.14 to `rhoai-operator`